scripts/e2e/datadog-logs/compose-temp-FU76eW.yaml

```diff
@@ -0,0 +1,72 @@
+services:
```

Comment:
I don't think you mean to commit this temp file. Maybe we should add it to `.gitignore`?